docs/tutorials/geomesa-examples-authorizations.rst

```diff
@@ -78,21 +78,7 @@ to show how authorizations work.
 Visibilities in GeoMesa
 -----------------------
 
-GeoMesa supports applying a single set of visibilities to all data in a
-``DataStore``. When configuring a ``DataStore``, the visibilities can be
-set with the ``geomesa.security.visibilities`` parameter:
-
-.. code-block:: java
-
-    // create a map containing initialization data for the GeoMesa data store
-    Map<String, String> configuration = new HashMap<>();
-    configuration.put("geomesa.security.visibilities", "user&admin");
```

Comment:
When did this option / capability go away?

Reply:
i got rid of it in this PR - i think there's not really a use-case for it anymore.